packages/office-ui-fabric-react/src/components/Icon/examples/Icon.SvgFactory.Example.tsx

```diff
@@ -1,24 +1,74 @@
 import * as React from 'react';
-import { OneDriveIcon, YammerIcon, BorderBlindsIcon } from '@fluentui/react-icons';
-import { mergeStyles } from 'office-ui-fabric-react/lib/Styling';
+import * as ReactIcons from '@fluentui/react-icons';
+import { PrimaryButton } from 'office-ui-fabric-react/lib/Button';
+import { mergeStyleSets } from 'office-ui-fabric-react/lib/Styling';
 
-const rootClass = mergeStyles({
-  display: 'flex',
-  selectors: {
-    '> *': {
-      height: 50,
-      width: 50,
-      marginRight: 25,
+const classes = mergeStyleSets({
+  root: {
+    selectors: {
+      '> *': {
+        height: 50,
+        width: 50,
+        marginRight: 25,
+      },
     },
   },
+  cell: {
+    display: 'flex',
+    flexDirection: 'column',
+    alignItems: 'center',
+    margin: '80px',
+    float: 'left',
+  },
+  code: {
+    background: '#f2f2f2',
+    borderRadius: '4px',
+    padding: '4px',
+  },
+  navigationText: {
+    width: 100,
+    margin: '0 5px',
+  },
 });
 
+const icons = Object.keys(ReactIcons).reduce((acc: React.FC[], exportName) => {
+  if (!!(ReactIcons as any)[exportName].displayName) {
+    acc.push((ReactIcons as any)[exportName] as React.FC);
+  }
+
+  return acc;
+}, []);
+
+const numOfIcons = icons.length;
+const numOfPages = parseInt((numOfIcons / 100).toString(), 10) + 1;
```

Comment:
what if there are 100 icons? are we getting 2 pages?

Reply:
Fixed